RebelAllianceBank/Users/Customer.cs

```diff
@@ -150,7 +162,7 @@ public void TransferUserToUser(List<IUser> users)
                 }
             }
 
-            while (otherAccount == null)
+            while (otherAccount != null)
```

Comment:
Är inte detta fel? Betyder inte detta att while loopen bara körs när otherAccount inte är lika med null? Då körs den väl aldrig?

Reply:
No, because otherAccount starts at null and in the while loop we set either null or a user on row 155